packages/core/src/browser/view-container.ts

```diff
@@ -46,6 +46,11 @@ export interface ViewContainerTitleOptions {
     closeable?: boolean;
 }
 
+enum SashType {
+    topSash = 'top-sash',
+    bottomSash = 'bottom-sash'
```

Comment:
minor, [use PascalCase](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#pascalcase-enum) for enum vals